vm/parser.ts

```diff
@@ -711,6 +861,17 @@ module J2ME {
       }
     }
 
+    getClassNameSlow(): string {
+      return this._name || (this._name = ByteStream.readString(this.utf8Name));
+    }
```

Comment:
> That's a worthy concern, but this changeset would be even smaller if you didn't replace all the className references with getClassNameSlow calls.

True. I was intending to change name, signature, implKey and all the other getters to getXXXSlow() to discourage use, but I postponed that for a future PR.